modules/ocf_tv/files/i3/config

```diff
@@ -69,14 +74,14 @@ bindsym $mod+Shift+Right move right
 bindsym $mod+semicolon split h
 
 # split in vertical orientation
-bindsym $mod+v split v
+bindsym $mod+slash split v
```

Comment:
I'd hesitate from changing some of the core keybindings away from the defaults, it will interfere a lot with people's muscle memory.

Reply:
mod+semicolon is not a default and was already changed. mod+semicolon together with mod+v has to be the stupidest thing I've ever seen.

Reply:
mod+semicolon is there because that's what was in nickimp's config when he shipped this. I'd advocate removing it and returning to the defaults.